.github/workflows/release.yaml

```diff
@@ -28,14 +34,19 @@ jobs:
           key: cache-${{ env.cache-version }}-go-${{ env.go-version }}-${{ hashFiles('v2/Makefile') }}
       - run: make setup
         if: steps.cache-tools.outputs.cache-hit != 'true'
-      - run: make image
-      - name: Login to ghcr.io
-        run: echo ${{ secrets.GITHUB_TOKEN }} | docker login ghcr.io -u $GITHUB_ACTOR --password-stdin
-      - name: Push to ghcr.io
-        run: |
-          IMAGE_TAG=${GITHUB_REF#refs/tags/v} # Remove "v" prefix.
-          docker tag coil:dev ghcr.io/cybozu-go/coil:$IMAGE_TAG
-          docker push ghcr.io/cybozu-go/coil:$IMAGE_TAG
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v2
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+      - name: Build and push
+        uses: docker/build-push-action@v3
+        with:
+          context: .
+          platforms: linux/amd64,linux/arm64/v8
+          push: true
+          tags: ghcr.io/cybozu-go/coil:${GITHUB_REF#refs/tags/v} # Remove "v" prefix.
```

Comment:
`${GITHUB_REF#/refs/tags/v}` seems not to be able to reference in `docker/build-push-action`.
So, we need the other way.

For example, we can use `output parameter`.

https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-an-output-parameter